src/wattsi.pas

```diff
@@ -1972,10 +1975,12 @@ procedure PreProcessCanIUseData(const CanIUseJSONFilename: AnsiString);
    States: set of TImplState;
    NewState: TImplGoodState;
 begin
-   Writeln('Parsing caniuse.com data...');
+   if (not(Quiet)) then
+      Writeln('Parsing caniuse.com data...');
```

Comment:
> I don't know this language, but isn't it possible to define a Log helper that does nothing if not quiet?

Yeah, almost certainly it is. I don’t know the language either, so this was just my first attempt at hacking in something that would work. I would be glad to iterate on it further if/when Hixie has time to provide some guidance (or if he time to code it up himself if he reckons that would be faster).